client/src/utils/curriculum-helpers.js

```diff
@@ -1,28 +1,17 @@
-import { parse } from '@babel/parser';
-import generate from '@babel/generator';
-
-const removeHtmlComments = str => str.replace(/<!--[\s\S]*?(-->|$)/g, '');
-
-const removeCssComments = str => str.replace(/\/\*[\s\S]+?\*\//g, '');
+import strip from 'strip-comments';
 
 export const removeJSComments = codeStr => {
-  // Note: removes trailing new lines and tailing spaces at end of lines
-  const options = {
-    comments: false,
-    retainLines: true,
-    compact: false,
-    concise: false,
-    minified: false
-  };
   try {
-    const ast = parse(codeStr);
-    const { code } = generate(ast, options, codeStr);
-    return code;
+    return strip(codeStr);
   } catch (err) {
     return codeStr;
   }
 };
 
+const removeHtmlComments = str => str.replace(/<!--[\s\S]*?(-->|$)/g, '');
+
+const removeCssComments = str => str.replace(/\/\*[\s\S]+?\*\//g, '');
+
```

Comment:
So the lib says it supports HTML and CSS - I wonder if it's worth using it for these helpers too.

Reply:
It does. If I remember correctly it requires you to specify it using an option.

`strip(html, { language: 'html'})`

I also didn't really test it much with CSS/HTML though. But it might be worth looking into.